src/market/market_utils.cairo

```diff
@@ -497,7 +497,6 @@ fn get_max_open_interest(
 /// * `delta` - The amount to increment by.
 fn increment_claimable_collateral_amount(
     data_store: IDataStoreDispatcher,
-    chain: IChainDispatcher,
     event_emitter: IEventEmitterDispatcher,
     market_address: ContractAddress,
     token: ContractAddress,
```

Comment:
If you wanna go with the `market_address` change the other to `XXX_address` or do the other way. Do that for the rest of the functions too please